operations/aws-security-roles/eis-security-audit-trusting-role.yml

```diff
@@ -2,7 +2,8 @@ AWSTemplateFormatVersion: 2010-09-09
 Description: Mozilla Enterprise Information Security (EIS) Security Audit Role
 Metadata:
   Source: https://github.com/mozilla/security/tree/master/operations/aws-security-roles/eis-security-audit-trusting-role
-  TemplateVersion: 3.0.0
+  URL: https://s3.us-west-2.amazonaws.com/public.us-west-2.infosec.mozilla.org/infosec-security-roles/cf/eis-security-audit-trusting-role.yml
```

Comment:
I like that we have Metadata that includes the URL (assuming it makes it easier for us to reference in Cloudformation), but it would be nice if this object name had some versioning associated with it.

Reply:
I imagine we could maintain both versioned files as well as a `-latest` file that we have people use when deploying. If accounts always reference a single s3 file (like they do now or like a `-latest` file) then we/they can just click the "update stack" button, leaving the URL pre-filled and they'll get the update. I'll think about this.